dsa-question-recommender/src/app/auth/register/page.js

```diff
@@ -0,0 +1,51 @@
+import GoogleIcon from '@mui/icons-material/Google'
+
+export default function Page () {
+  return (
+    <section className='flex flex-col items-center w-full'>
```

Comment:
Probably add some margin on the sides for mobile screens.
![image](https://github.com/GDGVIT/DSA-Question-Recommender-Frontend/assets/91245420/83789c69-f68e-467d-bf5a-bc5056ed0976)